class CreationError(Exception):
    pass


class OAuthError(Exception):
    pass
